### PR TITLE
table: Include method that clears all selected checkboxes

### PR DESCRIPTION
**Pull request by dimitrisnl**
Connects-to: #560
Change-type: minor
Signed-off-by: Dimitrios Lytras <dimitrios@resin.io>
---- Autogenerated Waffleboard Connection: Connects to #560

### CODE REVIEW

**Comment by dimitrisnl** [comment]
Pending
- Documentation
- Test

**Comment by dimitrisnl** [comment]
Enzyme with refs seems to be a bit tricky

**Comment by LucianBuzzo** [comment]
@dimitrisnl In enzyme you would just call the method directly I believe

File: src/stories/README/Table.md [hunk 1]
```diff
@@ -61,3 +61,8 @@ cause a re-render of the component and will not be reflected on the table.
 | `toBodyPrefix`    | <code>JSX.element &#124; JSX.Element</code> | - | - | JSX element(s) to display at the top of the table body |
 | `highlightedRows`    | <code>&ast;[]</code> | - | - | Highlights one or more rows. This property requires that you have provided a `rowKey` property: the row with a `rowKey` property that matches one of these values is highlighted. 
 
+## Programmatically selecting rows
+
+The component has a `setRowSelection` method that can be accesssed via <a href='https://reactjs.org/docs/refs-and-the-dom.html' target='_blank' rel='noopener'>ref</a>.
+
+It will accept an array of rows `T[]`, or an empty array in order to clear the selection.
```
Comment (LucianBuzzo) [inline src/stories/README/Table.md]: Can you mention that the `rowKey` property is required?

**Comment by dimitrisnl** [comment]
@brownjohnf Any idea on why the reviewers check has stuck?

**Comment by dimitrisnl** [comment]
@LucianBuzzo There is a weird bug with the check about reviewer approvals. I'll request a review again if you don't mind hoping that this will fix it.

**Comment by dimitrisnl** [comment]
@resin-ci retest

File: src/stories/README/Table.md [hunk 2]
```diff
@@ -61,3 +61,10 @@ cause a re-render of the component and will not be reflected on the table.
...
+
+This property requires that you have provided a `rowKey` property.
```
Comment (LucianBuzzo) [inline src/stories/README/Table.md]: NIT can you change "property" to "method" here?

**Comment by mikesimos** [comment]
@resin-ci retest

**Comment by dimitrisnl** [comment]
@mikesimos Hmm, retest doesn't seem to do much

**Comment by dimitrisnl** [comment]
@resin-ci retest

**Pull request merged**